src/GafferUI/StandardNodeGadget.cpp

```diff
@@ -726,7 +726,7 @@ void StandardNodeGadget::renderLayer( Layer layer, const Style *style, RenderRea
 
 			break;
 		}
-		case GraphLayer::Overlay :
+		case GraphLayer::Highlighting :
```

Comment:
This isn't actually sufficient to stop some strikethroughs going on top of the focus gadget - because they are in the same layer, it comes down to which is drawn last. But that's the same as when they were both in the Highlighting layer, so I think that's fine...